src/index.ts

```diff
@@ -1,6 +1,25 @@
-const main = () => {
-    const message: string = "Hello Businext!";
-    console.log(message);
-}
+import { ApolloServer, gql } from 'apollo-server';
 
-main();
+const server_port_default: number = 4000;
+const server_port: number = parseInt(process.env.ENDPOINT_PORT || '') || server_port_default;
```

Comment:
This is relying on what `parseInt('')` returns.

```suggestion
const server_port: number = process.env.ENDPOINT_PORT ? parseInt(process.env.ENDPOINT_PORT) : server_port_default;
```